components/collapsible-panel/collapsible-panel-summary-item.js

```diff
@@ -0,0 +1,42 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit';
+import { bodySmallStyles } from '../typography/styles.js';
+
+/**
+ * A component for a "summary item" child component that describes the content in a collapsible panel.
+ */
+class CollapsiblePanelSummaryItem extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: Text that is displayed
+			 * @type {string}
+			 */
+			text: { type: String },
+		};
+	}
+
+	static get styles() {
+		return [bodySmallStyles, css`
+			:host {
+				color: var(--d2l-color-galena);
+				display: block;
+			}
```

Comment:
If you override the default `display`, you need to also add:

```css
host([hidden]) {
  display: none;
}
```

[More details](https://daylight.d2l.dev/developing/best-practices/styles/#do%3A-style-the-%22display%22-value-of-the-host).

Reply:
This comment was marked as resolved, but there doesn't appear to be any styles for this. Did we decide to exclude them for some reason?

Reply:
Oh good catch. I added these styles in 8d86e8d8725e260ce85254e25b7b09d6a83fbdd1, but only to the collapsible panel (not the summary item). Will fix 👍